local/local_test.go

```diff
@@ -51,40 +51,53 @@ func testLocalImage(t *testing.T, when spec.G, it spec.S) {
 	})
 
 	when("#NewImage", func() {
-		when("image is available locally", func() {
-			var repoName = newTestImageName()
+		when("no base image is given", func() {
+			it("returns an empty image", func() {
+				_, err := local.NewImage(newTestImageName(), dockerClient)
+				h.AssertNil(t, err)
+			})
 
-			it.After(func() {
-				h.AssertNil(t, h.DockerRmi(dockerClient, repoName))
+			it("sets sensible defaults for all required fields", func() {
+				// os, architecture, and rootfs are required per https://github.com/opencontainers/image-spec/blob/master/config.md
+				img, err := local.NewImage(newTestImageName(), dockerClient)
+				h.AssertNil(t, err)
+				h.AssertNil(t, img.Save())
+				defer h.DockerRmi(dockerClient, img.Name())
+				inspect, _, err := dockerClient.ImageInspectWithRaw(context.TODO(), img.Name())
+				h.AssertNil(t, err)
+				h.AssertEq(t, inspect.Os, "linux")
+				h.AssertEq(t, inspect.Architecture, "amd64")
+				h.AssertEq(t, inspect.RootFS.Type, "layers")
 			})
+		})
 
-			it("returns the local image", func() {
-				labels := make(map[string]string)
-				labels["some.label"] = "some.value"
+		when("#FromBaseImage", func() {
```

Comment:
This looks to be accidentally nested under `#NewImage`

Reply:
That is because `FromBaseImage` is a option that is provided to `NewImage`